keepalive.go

```diff
@@ -54,6 +54,10 @@ func (cli *Client) keepAliveLoop(ctx context.Context) {
 					go cli.dispatchEvent(&events.KeepAliveRestored{})
 				}
 				lastSuccess = time.Now()
+				go cli.dispatchEvent(&events.KeepAliveSuccess{
+					Jid:         cli.Store.ID,
+					LastSuccess: lastSuccess,
+				})
```

Comment:
What's the point of this? There are already events for keepalive failing and restoring